packages/eslint-plugin/src/rules/interface-name-prefix.ts

```diff
@@ -10,6 +10,8 @@ export default util.createRule<Options, MessageIds>({
     docs: {
       description: 'Require that interface names be prefixed with `I`',
```

Comment:
Can we update this description while we are touching this?

The combo of the current description and your new comment actually tripped me up for a second and I thought you were saying we require that interfaces _are_ prefixed with an `I`. I think we should just update the description to reflect the fact that it is either require _or_ ban